OJP/OJP_JourneySupport.xsd

```diff
@@ -57,6 +57,32 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="ServiceSectionStructure">
+		<xs:annotation>
+			<xs:documentation>Properties of a service combined with a stop sequence interval. In many cases the elements about VEHICLE JOURNEY etc that are bundled in a service in OJP are enough for the whole Leg. However, in some cases enough of these element change within a single leg (usually a PT RIDE LEG), that e.g. the LINE, OPERATOR, FARE SECTION, COMMON SECTION changes. Possible cases are splitting/joining of trans, international trains at the border. Then it becomes necessary to tell at a StopPoint that the information has changed and this results in multiple ServiceSections. This implementation of ServiceSection is a view of some parts of JOURNEY PARTs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:group ref="StopSequenceIntervalGroup"/>
+			<xs:group ref="ServiceGroup"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:group name="StopSequenceIntervalGroup">
+		<xs:annotation>
+			<xs:documentation>Interval of stop sequence numbers.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="FromStopSeqNumber" type="xs:positiveInteger" minOccurs="0">
```

Comment:
I guess this is a numbering within the Leg (not with the VJ) where 0 (not 1) is the number of the boarding stop ?

Reply:
As it is a positiveInteger, my guess would be that, in case of a leg, 1 would be the boarding stop...

Reply:
It starts with 1 and 1 is the boarding station  I update the comments.